### PR TITLE
Add QueryType for Version on Assets 

### PR DESCRIPTION
**Pull request by leitge**

### CODE REVIEW

File: src/GraphQL/Resolver/AssetType.php [hunk 1]
```diff
@@ -363,6 +363,20 @@ public function resolveDuration(ElementDescriptor | null $value = null, array $c
         return $asset->getDuration();
     }
 
+    public function resolveVersion($value = null, $args = [], $context = [], ResolveInfo $resolveInfo = null)
```
Comment (Corepex) [inline src/GraphQL/Resolver/AssetType.php:369]:
```suggestion
     /**
     * @throws Exception
     */
    public function resolveVersion($value = null, $args = [], $context = [], ResolveInfo $resolveInfo = null)
```

**Review by Corepex — CHANGES_REQUESTED** [review]
@leitge, thanks for your PR. 
Tested it - looks good to me :) just one (very picky) thing to make the IDE happy :rofl:

**Comment by Corepex** [comment]
@leitge friendly reminder :alarm_clock: :lollipop:

**Review by leitge — COMMENTED** [review]
adjusted

**Comment by leitge** [comment]
@Corepex done ;)

**Review by Corepex — APPROVED** [review]
LGTM - thanks @leitge for your PR. 
Great to actively work with you on data-hub :100:

**Pull request merged**